regression-tests.recursor-dnssec/recursortests.py

```diff
@@ -521,6 +521,19 @@ def startAllAuth(cls, confdir):
                 ipaddress = cls._PREFIX + '.' + auth_suffix
                 cls.startAuth(authconfdir, ipaddress)
 
+    @classmethod
+    def waitForTCPSocket(cls, ipaddress, port):
+        for try_number in range(0, 100):
+            try:
+                sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+                sock.connect((ipaddress, port))
```

Comment:
I think a TCP connect on lo0 normally always gives a direct result (either refused or success) since things like  packet loss and other trouble are rare there, but ok.